src/Blocks/BlockNetherrack.h

```diff
@@ -0,0 +1,13 @@
+#pragma once
+
+
+#include "BlockHandler.h"
+
+class cBlockNetherrack : public cBlockHandler
+{
+public:
+	cBlockNetherrack(BLOCKTYPE a_Type) : cBlockHandler(a_Type){}
+
+	/** Checks whether the block has an effect on growing the plant */
+	virtual bool CanSustainPlant(BLOCKTYPE a_Plant) override { return (a_Plant == E_BLOCK_NETHER_WART); }
```

Comment:
Remove the doxy-comment (it should only go to the base class's function - so that there is just one version to maintain)